### ISSUE TITLE
Thumbnail: On accessing the image button as highlighted, speech output is made as ‘Toggle’ button.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3825]
Description: On accessing the image button as highlighted, speech output is made as ‘Toggle’ button.
Also, the selected information is not conveyed for the selected items.

Expected:  UI type should be conveyed according to the role assigned to the UI. 
Selection information shall be conveyed along with the UI element.

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98773805-b681c780-240f-11eb-8d77-2c18241bca5c.png)

**Comment by puru-hk** [comment]
Tested it is working as expected.
